apps/src/levelbuilder/skills/AccuracyCheck.tsx

```diff
@@ -176,6 +197,14 @@ const AccuracyCheck: React.FC<{levelId: number}> = ({levelId}) => {
           disabled={aiEvaluatedAnswers.length === 0}
         />
       </div>
+      {aiEvaluatedAnswers.length > 0 &&
+        !evaluationPending &&
+        questionAiEvaluatable && (
+          <div style={{marginTop: '12px'}}>
+            🎉 Evaluation complete
+            <AccuracyDetails evaluations={aiEvaluatedAnswers} />
```

Comment:
I wonder if we want to show the % accuracy score outside the table and then click in via dropdown for the table

Reply:
I thought about that too - and actually had it that way, but realistically I expect LBs to want to see the analysis anyway... I'd perhaps put that into a "let's see what Erik says" category too.